src/use-browser.ts

```diff
@@ -9,12 +9,14 @@ type BrowserOptions = {
   seleniumType: string;
   browserCreatorOptions: Record<string, any>;
   webdriverOptions: Partial<WebDriverOptions>;
+  enableBrowserErrors: boolean;
```

Comment:
The best naming practice is to call properties in a way that default value is always false (you can see how most of our components work same way, with small exceptions). So this property should be `skipConsoleErrorsCheck`

other than the naming, good addition